group_vars/all.yml

```diff
@@ -1497,6 +1497,7 @@ varnish_strip_params:
   - "trk_[^=&]+"
   - "dm_i"
   - "fbclid"
+  - "_mageops_redirect_source"
```

Comment:
Why we need this param with such a strange name? 
Can't we just use ?ref= 

If not, something more meaningful like ?redirect_from=

Reply:
In the worst case - make this parametrized.